### PR TITLE
updates for CreateAsk based on Josh's comments

### PR DESCRIPTION
**Pull request by michaelneuder**

### CODE REVIEW

File: src/bridges/zora/ZoraBridge.sol [hunk 1]
```diff
@@ -155,26 +147,27 @@ contract ZoraBridge is BridgeBase {
...
                 revert ErrorLib.InvalidInputA();
             }
-            // Output type needs to be ETH.
-            if (_outputAssetA.assetType != AztecTypes.AztecAssetType.ETH) {
+            // Output type needs to be VIRTUAL.
+            if (_outputAssetA.assetType != AztecTypes.AztecAssetType.VIRTUAL) {
                 revert ErrorLib.InvalidOutputA();
             }
 
+            uint256 inputAssetId = _inputAssetA.id;
```
Comment (critesjosh) [inline src/bridges/zora/ZoraBridge.sol:155]: https://github.com/michaelneuder/aztec-connect-bridges/pull/12/files#r1065226502

File: src/bridges/zora/ZoraBridge.sol [hunk 2]
```diff
@@ -183,7 +176,13 @@ contract ZoraBridge is BridgeBase {
                 sellerFundsRecipient,
                 0                     // Leave the finder fee as 0.
             );
-            return (0, 0, false);
+            
+            // Update the asset map to correlate new virtual token with existing
+            // bridge-owned nft.
+            nftAssets[_interactionNonce] = token;
```
Comment (critesjosh) [inline src/bridges/zora/ZoraBridge.sol:182]: use `_updateVirtualAssetId` like you do with the cancel ask

Comment (michaelneuder) [reply src/bridges/zora/ZoraBridge.sol:182]: yup, updated!

File: src/bridges/zora/ZoraBridge.sol (hunk 1 above)
Comment (michaelneuder) [reply src/bridges/zora/ZoraBridge.sol:155]: i was getting a stack too deep error from using _inputAssetA.id every time. i guess it adds an element to the stack every time it accesses the .id field? I will look at it again to see if i can get rid of that error some other way.

**Pull request merged**